.github/workflows/git_tag_released_version.yml

```diff
@@ -32,7 +32,7 @@ on:
 
 jobs:
   generate_tags:
-    runs-on: [self-hosted, ubuntu-20.04]
+    runs-on: ubuntu-latest
```

Comment:
Its not totally clear if these release tasks will work on hosted runners (or if they'll work at all), but its probably fine to include them. I may try these out during my release

Reply:
I thought this was the bot that tagged finished issues/PRs with the release that includes them, and since a release just went out...

Reply:
No, that one is different - this is only run manually and can do one of the release steps (tagging a released version in GitHub) - https://github.com/apache/beam/blob/fac67922cad99979aa6bba5339616fd253e932a7/.github/workflows/git_tag_released_version.yml#L24 (workflow_dispatch means manual trigger only). You're thinking of https://github.com/apache/beam/blob/20c8d3a2ba11f3965022e2ba5e12a241a4ec3442/.github/workflows/assign_milestone.yml#L20

Reply:
Fortunately, that should be functioning fine since its on a hosted runner